app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixPriceRangeBottomSheet.kt

```diff
@@ -42,7 +42,9 @@ fun QuickFixPriceRangeBottomSheet(
     onApplyClick: (Int, Int) -> Unit,
     onDismissRequest: () -> Unit,
     onClearClick: () -> Unit,
-    clearEnabled: Boolean
+    clearEnabled: Boolean,
+    start: Int = 500,
+    end: Int = 2500
```

Comment:
Could you be so nice as to explain to me what these values are supposed to represent ? Thanks 😄

Reply:
These are values I added to keep track of the circles and tooltips, specifically made for remembering the value of the last time u applied the filter.